enterprise/server/remote_execution/filecache/filecache.go

```diff
@@ -276,15 +276,19 @@ func key(ctx context.Context, node *repb.FileNode) string {
 	return k
 }
 
+var requestCountMetric map[bool]prometheus.Counter
+
+func init() {
+	requestCountMetric = make(map[bool]prometheus.Counter, 2)
```

Comment:
i'd initialize these metrics in the same place as the other metrics are initialized but it's not a big deal if you want to keep them here